receiver/sqlqueryreceiver/integration_test.go

```diff
@@ -171,7 +171,10 @@ func testMovieMetrics(t *testing.T, rm pmetric.ResourceMetrics) {
 
 	for _, metric := range metricsByName["genre.count"] {
 		pt := metric.Gauge().DataPoints().At(0)
-		genre, _ := pt.Attributes().Get("genre")
+		genre, exists := pt.Attributes().Get("genre")
+		if !exists {
```

Comment:
What do you think about instead of this, passing in the name of the column, or perhaps an isOracle boolean?

Reply:
Passing in the name of the key should work, as that seems more generic. The current layout of the test requires it to be something genre specific (since the variable is called genre), but in case other drivers end up being case-sensitive as well, it makes sense to me to pass in the genre attribute key.